docs/src/main/sphinx/installation/query-resiliency.rst

```diff
@@ -0,0 +1,119 @@
+======================================================
+Improve query resiliency with fault-tolerant execution
+======================================================
+
+You can configure Trino to be more resilient against query failure
```

Comment:
query processing failures

Reply:
I think "failures during query processing" is best here, actually